core/src/test/resources/HistoryServerExpectations/completed_app_list_json_expectation.json

```diff
@@ -22,6 +23,7 @@
     "duration" : 101795,
     "sparkUser" : "jose",
     "completed" : true,
+    "appSparkVersion" : "",
```

Comment:
This doesn't look correct. If I look at the log file for this app (`core/src/test/resources/spark-events/app-20161115172038-0000`), there's a log start event:

```
{"Event":"SparkListenerLogStart","Spark Version":"2.1.0-SNAPSHOT"}
```

So this value should be `"2.1.0-SNAPSHOT"`, no?